build/README.md

```diff
@@ -6,3 +6,14 @@ PRs opened against the `mattermost-server` repository will use the file called `
 
 The `Dockerfile` in this folder (`Dockerfile.buildenv`) is the build environment for our current builds you can find the docker image to downalod [here](https://hub.docker.com/r/mattermost/mattermost-build-server/tags/) or build your own.
 
+
+
+### Docker Image for building the Server
+
+We have a docker image to build `mattermost-server` and it is based on Go docker image.
+
+In our Docker Hub Repository we have the following images:
+
+- `mattermost/mattermost-build-server:dec-7-2018` which is based on Go 1.11 you can use for MM versions <= `5.8.0`
```

Comment:
```suggestion
- `mattermost/mattermost-build-server:dec-7-2018` which is based on Go 1.11 you can use for MM versions <= `5.9.0`
```

Reply:
5.9 is not released yet, and if we merge this we will use 1.12 to release 5.9

Reply:
Won't v5.9 be branched off of v5.8? I think dependency upgrades are reserved for `master`, not stable releases as such.

Reply:
ahh, true